.travis.yml

```diff
@@ -0,0 +1,35 @@
+---
+sudo: required
+dist: trusty
+language: node_js
+
+node_js:
+  - 9
+python:
+  - 3.6.x
+
+addons:
+  apt:
+    packages:
+      - imagemagick
+      - graphicsmagick
+
+install:
+  - wget https://github.com/gohugoio/hugo/releases/download/v0.31.1/hugo_0.31.1_Linux-64bit.deb
+  - sudo dpkg -i hugo*.deb
+  - npm install
+  - npm install --global gulp-cli
+
+script:
+  - gulp build
+  - if [[ "$TRAVIS_TAG" != "" ]]; then echo 'Building tag artifact'; fi
```

Comment:
Can these be in one `if`?

Reply:
They could, but they are not. It is just a debug message, not a complex operation.

Reply:
Your call...